load_tests/script.js

```diff
@@ -23,8 +23,8 @@ export const options = {
     },
   },
 
-  vus: 1,
-  duration: "15s",
+  // vus: 1,
+  // duration: "15s",
 };
 
 const scorerId = 24;
```

Comment:
Can we parameterize scorerId as well? And set it to 24 in the k6 cloud? That way we can test locally with other keys without having to hardcode. Unless the intention is to just share the one set of keys for id 24

Reply:
Yes, looking into this